README.md

```diff
@@ -1,27 +1,21 @@
 # Course Planner
+Course Planner is a project created by students at the University of Auckland for the `SOFTENG761` course. The function of this application is to allow a user to create a customized degree 'plan' for students. This plan shows which courses the student would take per semester for their entire degree.
 
-<details><summary>  Setting up the development environment for MacOS and Linux </summary>
-<p>
+#  Setting up the development environment
 
-Prerequisities: Docker
+### MacOS and Linux
+> Prerequisities: `Docker`
 
 1. Install [Docker Desktop](https://docs.docker.com/docker-for-mac/install/)
 2. Start Docker desktop
 3. Navigate to base repository `course-planner`
 4. Open up terminal/command-line and run `docker-compose up --build`. Note that if the images have already been built then you can run `docker-compose up`
 5. Wait until everything starts up
-6. To test the api is running, go to [localhost:8080/heartbeat](localhost:8080/heartbeat)
-7. To test the frontend is running, go to [localhost:3000](localhost:3000)
+6. To test the api is running, go to [localhost:8080/heartbeat](http://localhost:8080/heartbeat)
+7. To test the frontend is running, go to [localhost:3000](http://localhost:3000)
 
-Once you are done CTRL+C and then run the command: `docker-compose down` to stop containers.
-
-</p>
-</details>
-
-<details><summary>  Setting up the development environment for Windows </summary>
-<p>
-
-Prerequisities: Docker, Windows 10 Education or Professional, dos2unix.exe
+### Windows
+> Prerequisities: `Docker`, `Windows 10 Education or Professional`, `dos2unix.exe`
```

Comment:
version 1903/1909 now has wsl2 backport so any windows is possible assuming you have 1903 onwards with latest wsl update